test_programs/execution_success/bit_shifts_runtime/src/main.nr

```diff
@@ -17,4 +17,6 @@ fn main(x: u64, y: u8) {
     assert(a << y == -2);
 
     assert(x >> (x as u8) == 0);
+
+    assert_eq(a >> 3, -97);
```

Comment:
`a` here should be `-1` due to the `let mut a: i8 = y as i8; a = -a;` above.

Also, it looks like we can't have negatives in the Prover.toml unfortunately:
```
input file is badly formed, could not parse, TOML parse error at line 3, column 5
  |
3 | a = -769
  |     ^^^^
data did not match any variant of untagged enum TomlTypes
```

Reply:
We can have negative numbers in Prover.toml, with double quotes: "-769"
The 'a' was copy/pasted from another file where I was testing, I did not see it was already used here... I have renamed it.

Reply:
Looks like bit_shifts_runtime is still failing